lib/student.rb

```diff
@@ -1 +1,20 @@
 
+class Student
+  attr_reader :name, :age, :scores, :grade
```

Comment:
```suggestion
  attr_reader :name, :age, :scores
```

`grade` is not an instance variable so we don't need an `attr_reader` for it. You also define a method called `grade` on line 14 which conflicts with the method `grade` created by the `attr_reader`.